features/recipes/router.py

```diff
@@ -285,3 +286,59 @@ def delete_recipe(recipe_id: int, user: common.authentication.authenticated_user
             status_code=fastapi.status.HTTP_404_NOT_FOUND,
             detail=f"Recipe with {recipe_id=} does not exist",
         )
+
+
+@recipes_router.put('/{recipe_id}', response_model=RecipeResponse)
+def update_recipe(recipe_id: int = fastapi.Path(),
+                 update_recipe_input_model:  RecipeInputModel = fastapi.Body()):
+
+    """
+    Update recipe
+
+    :param recipe_id:
+    :param update_recipe_input_model:
+    :return:
+    """
+
+    try:
+        return features.recipes.operations.update_recipe(recipe_id=recipe_id)
```

Comment:
This function 'update_recipe' is created and used for something else. Check the code and validate if it is the proper way to do your task. You have the update the whole recipe model with completely new one